<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>439</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Linux&amp;Qt即时通讯</string>
  </property>
  <widget class="QTableWidget" name="userTableWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>310</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <column>
    <property name="text">
     <string>用户名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>主机名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>IP地址</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
  </widget>
  <widget class="QTextBrowser" name="messageBrowser">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>391</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="messageTextEdit">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>312</y>
     <width>391</width>
     <height>96</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>加粗</string>
   </property>
  </widget>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>289</y>
     <width>112</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="sizeComboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>289</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
  </widget>
  <widget class="QToolButton" name="sendFileBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>传输文件</string>
   </property>
   <property name="text">
    <string>send</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/send.png</normaloff>:/images/send.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="saveToolBtn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>save chat history</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>340</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>手动设置广播地址</string>
   </property>
   <property name="text">
    <string>手动设置局域网群聊广播地址</string>
   </property>
  </widget>
  <widget class="QLabel" name="userNumLabel">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>300</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>在线用户:0人</string>
   </property>
  </widget>
  <widget class="QLabel" name="webIPLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>外网IP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>410</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkGroupBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>380</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>局域网群聊</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>410</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addContactBtn">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>340</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>添加联系人</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeContactBtn">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>370</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>移除联系人</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readChatBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>打开消息记录浏览器</string>
   </property>
   <property name="text">
    <string>聊天记录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>380</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>300</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>联系人列表：</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QToolButton" name="boldBtn">
    <property name="toolTip">
     <string>加粗</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/bold.png</normaloff>:/images/bold.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="italicBtn">
    <property name="toolTip">
     <string>斜体</string>
    </property>
    <property name="text">
     <string>斜体</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/italic.png</normaloff>:/images/italic.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="underlineBtn">
    <property name="toolTip">
     <string>下划线</string>
    </property>
    <property name="text">
     <string>下划线</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/under.png</normaloff>:/images/under.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="colorlBtn">
    <property name="toolTip">
     <string>更改字体颜色</string>
    </property>
    <property name="text">
     <string>更改颜色</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/color.png</normaloff>:/images/color.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <zorder>userTableWidget</zorder>
  <zorder>messageBrowser</zorder>
  <zorder>messageTextEdit</zorder>
  <zorder>fontComboBox</zorder>
  <zorder>sizeComboBox</zorder>
  <zorder>colorlBtn</zorder>
  <zorder>saveToolBtn</zorder>
  <zorder>lineEdit</zorder>
  <zorder>userNumLabel</zorder>
  <zorder>webIPLabel</zorder>
  <zorder>sendButton</zorder>
  <zorder>checkGroupBox</zorder>
  <zorder>closeButton</zorder>
  <zorder>addContactBtn</zorder>
  <zorder>removeContactBtn</zorder>
  <zorder>readChatBtn</zorder>
  <zorder>pushButton</zorder>
  <zorder>sendFileBtn</zorder>
  <zorder>label</zorder>
  <zorder>boldBtn</zorder>
  <zorder>italicBtn</zorder>
  <zorder>underlineBtn</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
